src/main/resources/click.js

```diff
@@ -0,0 +1,25 @@
+(function (element, offsetX, offsetY) {
+  const rect = element.getBoundingClientRect();
+
+  function mouseEvent() {
+    if (typeof (Event) === 'function') {
+      return new MouseEvent('click', {
+        'view': window,
+        'bubbles': true,
+        'cancelable': true,
+        'clientX': rect.left + rect.width / 2 + offsetX,
+        'clientY': rect.top + rect.height / 2 + offsetY
+      });
+    }
+    else {
+      const event = document.createEvent('MouseEvent');
```

Comment:
could you please clarify? :) is this JS executed by browser or something else? 
if by browser (and since this is meant to fix issue in IE11) - IE11 (and some other potato/vintage browsers) does not support declaring variables via `const` and `let`

Reply:
IE11 supports `const`. See https://caniuse.com/?search=const

Reply:
ups, probably it would be safer to use `var` instead of `const`. :)

But I tested this code with IE11, and it worked. Do you know some browsers that are used for testing via webdriver and don't support `const`?

Reply:
Tested on IE11 that's (unfortunately) installed on my corporate PC, version: 
![image](https://user-images.githubusercontent.com/6271070/109020562-bc4fa000-76c2-11eb-9a69-9ad89cad3d9a.png)
results:
![image](https://user-images.githubusercontent.com/6271070/109020386-975b2d00-76c2-11eb-8b2c-8076f1eee0e5.png)
then again - maybe that's just the browser's console that's just lying :) but most probably using the old fashioned `var` would've been safer :)
However if the existing solution fixes OP's problem then never mind ;)

Reply:
But does `$("#mylocator").click(ClickOptions.usingJavaScript());` work in your IE?

Reply:
ok, most probably it's indeed related to how IE11's console treats variable scope :) further tests revealed that if block of code is run as multiple lines, then the result is as expected:
![image](https://user-images.githubusercontent.com/6271070/109025008-ffac0d80-76c6-11eb-9031-0c34ba7f292b.png)
Thus I assume that the `click(usingJavaScript());` will indeed work in my IE :) Sorry to have bothered you :)